test/e2e/ingress.go

```diff
@@ -619,7 +619,15 @@ var _____ = framework.KubeDescribe("Ingress tests simple NLB", func() {
 		replicas := int32(3)
 		targetPort := 9090
 		backendContent := "mytest"
-		route := fmt.Sprintf(`* -> inlineContent("%s") -> <shunt>`, backendContent)
+		route := fmt.Sprintf(`*
+			-> setResponseHeader("Y-Host", "${request.host}")
+			-> setResponseHeader("Y-X-Forwarded-For", "${request.header.X-Forwarded-For}")
+			-> setResponseHeader("Y-X-Forwarded-Proto", "${request.header.X-Forwarded-Proto}")
+			-> setResponseHeader("Y-X-Forwarded-Port", "${request.header.X-Forwarded-Port}")
+			-> inlineContent("%s")
+			-> <shunt>`,
+			backendContent)
```

Comment:
these setResponseHeader templates are not evaluated for whatever reason. I tested inside the pod I get the same result and I compared the routes via ` cat /proc/1/cmdline | tr '\0' ' ' | hexdump -C` and they seem to be binary compatible with something I create on my desk and on my desk works as expected.

Reply:
Btw. skipper version has not the templating yet :)